project/Dependencies.scala

```diff
@@ -46,26 +46,22 @@ object Dependencies {
   val metricsStatsd: ModuleID =      "com.readytalk"         %  "metrics3-statsd"  % "4.2.0"
 
   val scalaLogging: ModuleID =    "com.typesafe.scala-logging"    %% "scala-logging"        % "3.1.0"
-  val jacksonCore: ModuleID =     "com.fasterxml.jackson.core"    % "jackson-core"          % "2.4.3"
+  val jacksonCore: ModuleID =     "com.fasterxml.jackson.core"    % "jackson-core"          % "2.8.10"
   val jodaTime: ModuleID =        "joda-time"                     % "joda-time"             % "2.9.4"
   val jodaConvert: ModuleID =     "org.joda"                      % "joda-convert"          % "1.8"
   val typesafeConfig: ModuleID =  "com.typesafe"                  % "config"                % "1.3.0"
   val ravenLogback: ModuleID =    "com.getsentry.raven"           % "raven-logback"         % "7.8.6"
   val swaggerUI: ModuleID =       "org.webjars"                   % "swagger-ui"            % "2.1.1"
   val commonsJEXL: ModuleID =     "org.apache.commons"            % "commons-jexl"          % "2.1.1"
   val cats: ModuleID =            "org.typelevel"                 %% "cats"                 % "0.9.0"
+  val parserCombinators =         "org.scala-lang.modules"        %% "scala-parser-combinators" % "1.0.6"
   val mysqlConnector: ModuleID =  "mysql"                         % "mysql-connector-java"  % "5.1.38"
   val liquibaseCore: ModuleID =   "org.liquibase"                 % "liquibase-core"        % "3.5.3"
-  val logbackClassic: ModuleID =  "ch.qos.logback"                % "logback-classic"       % "1.1.6"
+  val logbackClassic: ModuleID =  "ch.qos.logback"                % "logback-classic"       % "1.2.2"
   val scalatest: ModuleID =       "org.scalatest"                 %% "scalatest"            % "3.0.1" % "test"
   val mockito: ModuleID =         "org.mockito"                   % "mockito-core"          % "2.7.22" % "test"
   val mockserverNetty: ModuleID = "org.mock-server"               % "mockserver-netty"      % "3.9.2" % "test"
 
-  val spraySwagger: ModuleID = ("com.gettyimages" %% "spray-swagger" % "0.5.0"
-    exclude("com.typesafe.scala-logging", "scala-logging-slf4j_2.11")
-    exclude("com.typesafe.scala-logging", "scala-logging-api_2.11")
-    exclude("com.google.guava", "guava"))
-
```

Comment:
We didn't actually use spray-swagger anywhere. BUT, we did rely on the fact that spray-swagger included the scala parsers package. I've eliminated spray-swagger (since it also brought in vulnerable libraries) and replaced it with the `scala-parser-combinators` that we do need.